app/hotels/src/allHotels/AllHotelsSearchList.js

```diff
@@ -2,38 +2,102 @@
 
 import * as React from 'react';
 import idx from 'idx';
-import { createFragmentContainer, graphql } from 'react-relay';
-import { ScrollView, Text } from 'react-native';
-import { CenteredView, Logger } from '@kiwicom/react-native-app-shared';
+import { createPaginationContainer, graphql } from 'react-relay';
+import type { Relay } from 'react-relay';
```

Comment:
Are you sure this type actually exists? It seems that I can replace the props with this:

```diff
- relay: Relay,
+ relay: any,
```

and it still works. This is what actually happens when you import non-existent type. I would replace it with something more obvious if it's intentional:

```diff
- relay: Relay,
+ relay: $FlowFixMeProps,
```

Reply:
I think this should have been `RelayPaginationProp`

Reply:
Same problem, is it? Relay is not exporting Flow types as far as I know. From my point of view, this is still effectively `any` type.

Reply:
Well easy to verify that types doesn't work here:

```diff
- if (this.props.relay.hasMore() && !this.props.relay.isLoading()) {
+ if (this.props.relay.hasMore() && !this.props.relay.isLoadingShit()) {
```

```
▲ react-native-app at 249-hotels-pagination ✖ y flow
yarn run v1.3.2
$ /Users/mrtnzlml/Work/kiwi-opensource/react-native-app/node_modules/.bin/flow
No errors!
✨  Done in 0.21s.
```